app/errors.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+
+class VaccineError(Exception, ABC):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+
+    @abstractmethod
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "OutdatedVaccineError"
```

Comment:
The `__str__` method should provide a more descriptive error message. Consider returning a message like: `'Visitor's vaccine is expired.'`